src/map/battle.cpp

```diff
@@ -7360,6 +7358,16 @@ static void battle_calc_weapon_final_atk_modifiers(struct Damage* wd, struct blo
 			status_change_end(target, SC_REJECTSWORD);
 	}
 
+	// Poison React Envenom Level 5 Autocast
+	if (tsc != nullptr && wd->damage > 0) {
```

Comment:
Quite a few tsc checks.
Maybe we should combine all of them in the future.

Reply:
Yeah I didn't want the PR to affect hundreds of lines. And we eventually need to move these effects into the right order anyway, was thinking of doing that at the same time then.